modules/kamihaya_cms_auth/src/Routing/KamihayaRouteSubscriber.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Drupal\kamihaya_cms_auth\Routing;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Listens to the dynamic route events.
+ */
+class KamihayaRouteSubscriber extends RouteSubscriberBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function alterRoutes(RouteCollection $collection): void {
```

Comment:
@Masami-Suzuki522 san
Can we keep the routename user.login and just change the path to /kanrisha/login
E.g https://github.com/HappySeedIndia/takenet/blob/dev/docroot/modules/custom/kanrishalogin/src/Routing/RouteSubscriber.php